backend/go.mod

```
@@ -0,0 +1,5 @@
+module 2022_2_GoTo_team
```

Обычно именуют все же с путем до репы

```suggestion
module github.com/go-park-mail-ru/2022_2_GoTo_team
```

Соответственно, надо везде поменять